chutzpah-run.ps1

```diff
@@ -1,29 +1,72 @@
-﻿param (
-    [string]$nugetExe = ".\.nuget\NuGet.exe",
-    [string]$path,        # Adds a path to a folder or file to the list of test paths to run.
+﻿<#
+ #>
+param (
+    [string]$workingDir, # Optionally specify the working directory
+    [string]$nugetExe = ".\.nuget\NuGet.exe", # The path to the NuGet executable, only required if useExistingPackage is not set
+    [switch]$useExistingPackage, # Use the existing Chutzpah NuGet package, typically that was downloaded by version in an earlier build step
+    [string]$path,        # Adds semicolon-separated paths to folders or files to the list of test paths to run.
     [switch]$teamcity,    # Forces TeamCity mode (normally auto-detected)
     [switch]$coverage,    # Enable coverage collection
-    [switch]$wait,        # Wait for input after completion
```

Comment:
'wait' makes no sense in a teamcity job. I presume Jon added it merely because the console runner has it.